apps/condo/domains/news/schema/NewsItemTemplate.js

```diff
@@ -27,6 +28,19 @@ const NewsItemTemplate = new GQLListSchema('NewsItemTemplate', {
             kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
         },
 
+        name: {
+            schemaDoc: 'Name of template the news item',
```

Comment:
Add an example to all your schemaDocs. It should be obvious to user how to use this (User may be not engineer, but product manager)